_posts/2017-08-16-match-my-route-introduction.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Introduction to MatchMyRoute - an app for matching up experienced with inexperienced cyclists
+meta: Geovation does not only support project of their programme members, but we also work on our own projects. MatchMyRoute
+links into our current annual focus of greener and smarter communities. Its aim is to make cycling more accessible and lower the barrier to picking up cycling. This is done by connecting experienced cyclists who regularly take a route with inexperienced cyclists who can join the experienced on their route.
+author: Max Stephan
+comments: true
+permalink: matchmyroute
+---
+
+![MatchMyRoute Logo](/assets/matchmyroute_logo.png)
+
+# Introduction
+
+This post introduces a project at Geovation on lowering the barrier for people to pick up cycling. The idea is to create a platform for buddying up experienced with inexperienced cyclists so that they can cycle together. We are creating an app for achieving this task which will soon be available for iOS and Android.
+
+# TL/DR
+
+## What is the problem?
+
+* There are big barriers to cycling in cities, such as
+  * insufficient infrastructure (no (segregated) bike lanes)
+  * lack of confidence of how to behave in traffic
+  * complicated road network, difficult to figure out how to get from A to B
+  * pollution
+
+## How do we want to address this problem?
+
+* Bringing together experienced cyclists that regularly take a route with cycle freshers who would like to take a route
+* Providing a convenient mechanism for achieving the above and giving reward points as an incentive
+
+## What's the current status?
+
+* We have created an app and API providing the core functionality. This currently is in closed testing status but we're planning to go into public beta this year.
+
+## Register your interest
```

Comment:
I'd merge this with the section above, and make it more call-to-actiony... 'Want to hear first when it's in the app store? Sign up here and we'll let you know when we launch!' Or some such language